MonkeyWrench.Web.WebService/WebServices.asmx.cs

```diff
@@ -126,17 +138,19 @@ public void CreateLanefile (WebServiceLogin login, int lane_id, string filename)
 				throw new ArgumentException ("lane_id");
 
 			using (DB db = new DB ()) {
-				VerifyUserInRole (db, login, Roles.Administrator);
-				db.Audit (login, "WebServices.CreateLaneFile (lane_id: {0}, filename: {1})", lane_id, filename);
+				var lane = DBLane_Extensions.Create (db, lane_id);
+				VerifyUserInRoles (db, login, lane.additional_roles, false);
```

Comment:
This allows anybody to create the lanefile if `lane.additional_roles` is empty.

I think you need to create a list where you make sure Administrator is, and then add any additional roles to that list.